site2/website/releases.json

```diff
@@ -1,5 +1,6 @@
 [
   "2.4.2",
+  "2.5.0",
```

Comment:
nit: Put it at the first line?

Reply:
this was done intentionally. because we want algolia index the website first before making it the stable release.